src/tests/suite/daos_rebuild_simple.c

```diff
@@ -932,7 +932,7 @@ rebuild_small_pool_n4_setup(void **state)
 		 */
 		print_message("It can not create the pool with 4 ranks"
 			      " probably due to not enough ranks %d\n", rc);
-		return 0;
+		return rc;
```

Comment:
We should remove the comment above that says
```
		/* Let's skip for this case, since it is possible there
		 * is not enough ranks here.
		 */
```